x-pack/plugins/fleet/server/routes/package_policy/index.ts

```diff
@@ -52,6 +55,28 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
     getOnePackagePolicyHandler
   );
 
+  router.get(
```

Comment:
It would be good to cover these new endpoints with some functional tests and possibly document them in the openapi spec.

Reply:
I think API integration tests are probably a good idea here, but it might be best to leave these endpoints undocumented. This feels like a very "internal" endpoint for data recovery/cleanup, and we're planning on removing it anyway once a root cause of the orphaned policy issue is discovered and addressed in https://github.com/elastic/kibana/issues/134518.

Reply:
That's also fine. I heard in the demo that we plan to fix the bug rather than leaving this workaround in place, so it's ok to leave it undocumented.

Reply:
API tests added in https://github.com/elastic/kibana/pull/131322/commits/76abd735688e7298cb8029c576e619568741efac